src/app/queries/types/vms.types.ts

```diff
@@ -7,11 +7,12 @@ export interface IVMwareVMDisk {
 
 export interface IVMwareVMConcern {
   name: string;
-  severity: 'Warning' | 'Critical' | 'Advisory' | 'Info'; // TODO not sure if these are final
+  severity: 'Warning' | 'Critical' | 'Advisory' | 'Info' | 'Unknown';
```

Comment:
I don't think `'Unknown'` is a valid severity value in the API, but I'm not sure. We're not handling it above, either way, should probably remove it